app/views/about/accessibility.en.html.erb

```diff
@@ -0,0 +1,84 @@
+<% title 'Accessibility statement for Disclosure Checker' %>
+<div class="govuk-width-container">
+  <main class="govuk-main-wrapper"  id="main-content">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">Accessibility statement for Disclosure Checker</h1>
```

Comment:
I don't think we should publicly name this `Disclosure Checker`, that is more like an internal code-name, and instead use the name we have so far for the service `Check when a caution or conviction is spent` (in fact, the helper) as we do in the cookies or terms and conditions, `<%= service_name -%>`.
There are a few occurrences of this we can replace, including the title.

I've added a comment in Wes' document for clarification.